test-assets/7677a9/slider.js

```diff
@@ -0,0 +1,44 @@
+function increaseSlider() {
+    document.getElementById('motionSlider').value++;
+    document.getElementById('output').innerHTML = document.getElementById('motionSlider').value;
+}
+
+function decreaseSlider() {
+    document.getElementById('motionSlider').value--;
+    document.getElementById('output').innerHTML = document.getElementById('motionSlider').value;
+}
+
+function handleOrientation(event) {
+    if (event.gamma > 20) {
+        increaseSlider();
+    } else if (event.gamma < -20) {
+        decreaseSlider();
+    }
+}
+
+function handleOrientationCanBeDisabled(event) {
+    const disableMotion = document.getElementById('disableMotion');
+    const gamma = !disableMotion.checked ? event.gamma : 0;
```

Comment:
You can invert test outcomes to remove the negation
```suggestion
    const gamma = disableMotion.checked ? 0 : event.gamma;
```